### PR TITLE
[Update] add metadata keywords to the pages

### PR DESCRIPTION
**Pull request by Rajakavitha1**
# Description

add metadata to the pages that do not have the metadata

# Version

Which documentation version does this PR apply to?

- [* ] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes #[insert issue link, if any]

### CODE REVIEW

**Comment by Loquacity** [comment]
@Rajakavitha1 The tags and the keywords work differently. The tags drive the related content feature, but the keywords are just keywords that people can click on. It's better to have more metadata in the tags section, and not to overburden the keywords section (because it ends up presenting too many tags for people to click). Also, remember that order is important in the tags section: earlier tags have more 'weight' than later tags, and it's a maximum of 5-6 tags.

**Comment by Rajakavitha1** [comment]
> @Rajakavitha1 The tags and the keywords work differently. The tags drive the related content feature, but the keywords are just words that people can click on. It's better to have more metadata in the tags section, and not to overburden the keywords section (because it ends up presenting too many tags for people to click). Also, remember that order is important in the tags section: earlier tags have more 'weight' than later tags, and it's a maximum of 5-6 tags.
> 
> image

Noted!!! yes, makes sense.

File: timescaledb/page-index/page-index.js [hunk 1]
```diff
@@ -1,33 +1,69 @@
...
     href: 'timescaledb',
+    pageComponents:["featured-cards"],
+    tags: ["timescaledb", "overview", "get started", "how-to's", "contribute"],
+    keywords: ["TimescaleDB","get started","tutorials", "code", "contribute"],
     excerpt:
-      "All you need to know about TimescaleDB architecture, concepts, setup, how-to's, tutorials, and management",
+      "All you need to know about TimescaleDB architecture, concepts, setup, how-to's, tutorials, management, and contribution.",
+    filePath: "index.md",
     children: [
-      ...overviewPageIndex,
```
Comment (jacobprall) [inline timescaledb/page-index/page-index.js:15]: These imports are needed - without them the pages won't be generated under "TimescaleDB"

**Comment by Rajakavitha1** [comment]
Hi @Loquacity !!! I checked with @jacobprall  and he is working on automating the related content even if no metadata is provided. Could you please review this PR for the tagging and keywords.

File: timescaledb/page-index/page-index.js [hunk 2]
```diff
@@ -4,30 +4,72 @@ const howtoPageIndex = require('../how-to-guides/page-index/page-index')
...
 module.exports = [
   {
     title: 'TimescaleDB',
     href: 'timescaledb',
+    pageComponents:["featured-cards"],
+    tags: ["timescaledb", "overview", "get started", "learn", "contribute"],
+    keywords: ["TimescaleDB","get started","tutorials", "code", "contribute"],
     excerpt:
-      "All you need to know about TimescaleDB architecture, concepts, setup, how-to's, tutorials, and management",
+      "All you need to know about TimescaleDB architecture, concepts, setup, how-to's, tutorials, management, and contribution.",
```
Comment (Loquacity) [inline timescaledb/page-index/page-index.js:15]:
```suggestion
      "All you need to know about TimescaleDB architecture, concepts, setup, how-tos, tutorials, management, and contribution.",
```

File: timescaledb/page-index/page-index.js [hunk 3]
```diff
@@ -4,30 +4,72 @@ const howtoPageIndex = require('../how-to-guides/page-index/page-index')
...
         href: 'contribute-to-timescaledb',
+        tags: ["timescaledb", "overview", "get started", "learn"],
+        keywords: ["code", "GitHub", "git", "error"],
+        excerpt:"Learn more about how to contribute to TimescaleDB",
       },
       {
         title: 'Contribute to documentation',
         href: 'contribute-to-docs',
+        tags:["timescaledb", "get started", "learn", "contribute"],
+        keywords:["code", "docs", "markdown", "styleguide"],
```
Comment (Loquacity) [inline timescaledb/page-index/page-index.js:64]:
```suggestion
        keywords:["code", "docs", "markdown", "style guide"],
```

File: timescaledb/page-index/page-index.js [hunk 4]
```diff
@@ -4,30 +4,72 @@ const howtoPageIndex = require('../how-to-guides/page-index/page-index')
...
+        excerpt:"Learn more about how to contribute to TimescaleDB documentation",
       },
       {
         title: 'Compare TimescaleDB Editions',
         href: 'timescaledb-edition-comparison',
+        tags:["timescaledb", "get started", "learn", "contribute"],
+        keywords:["Apache", "community", "features",],
+        excerpt:"Compare different editions of TimecaleDB",
```
Comment (Loquacity) [inline timescaledb/page-index/page-index.js:72]:
```suggestion
        excerpt:"Compare different editions of TimescaleDB",
```

**Review by Loquacity — APPROVED** [review]
Some non-blocking suggestions, otherwise LGTM

**Comment by Rajakavitha1** [comment]
Incorporated the changes that @Loquacity  suggested. @charislam can you please take a look.

File: install/page-index/page-index.js [hunk 5]
```diff
@@ -4,17 +4,15 @@ module.exports = [
     href: "install",
     pageComponents: ["featured-cards"],
     tags: ["timescaledb", "install"],
-    keywords: ["install", "TimescaleDB"],
+    keywords: ["install", "TimescaleDB", "self hosted", "containers", "cloud images", "managed services"],
```
Comment (charislam) [inline install/page-index/page-index.js:7]:
```suggestion
    keywords: ["install", "TimescaleDB", "self-hosted", "containers", "cloud images", "managed services"],
```

**Review by charislam — APPROVED** [review]
LGTM!

**Review by Loquacity — APPROVED** [review]

**Review by jacobprall — APPROVED** [review]

**Pull request merged**